www/.env.development

```diff
@@ -1,2 +1,2 @@
 FEATURE_DIRECTION_SWITCHER=true
-SEGMENT_KEY=0123456789
+SEGMENT_KEY=
```

Comment:
nit: default to an empty string

Reply:
leaving as it is, adding conditional plugin connection